REQUIRE

```diff
@@ -1,2 +1,3 @@
 julia 0.6
 ImageCore
+ImageFiltering
```

Comment:
We can't afford ImageFiltering as a dependency, we had removed it in the past.

Reply:
I didn't notice that ImageFiltering is broken in nightly. I will process the image borders by myself.